### ISSUE TITLE
[a11y]: InlineLoading [playground] - status changes not conveyed

### ISSUE THREAD

**Issue by andreancardona** [carbon-design-system/carbon#14822]
### Package

@carbon/react

### Browser

Safari & iPhone

### Package version

@carbon/react@1.38.0

### Automated testing tool and ruleset

Manual QA

### Assistive technology

VO

### Description
Finished and error statuses are not programmatically conveyed

Screen reader does not identify the mentioned statuses hence the user would not be aware of it

### Recommended Fix
The current status that is shown using the Color/Visual icon etc. should be conveyed through appropriate text and programmatically associated as well. 

If the screen reader does not announce the status change in a page element, it’s difficult for users to understand it on their own. This defect mainly impacts users who are blind. 

### WCAG 2.1 Violation

4.1.3 Status Messages

**Comment by heloiselui** [comment]
Since this playground example doesn’t exist in our Storybook (@carbon/react v1.84), we’re not able to reproduce or verify the issue as described. I believe we can close this issue.
